From e0c9115e1ceb6621d6c04ae8bfd423a0452fea9c Mon Sep 17 00:00:00 2001
From: Christian Marangi <ansuelsmth@gmail.com>
Date: Wed, 5 Jul 2023 11:03:40 +0200
Subject: [PATCH] mreceive: msend: fix wrong version in -v output

-v output was never changed to follow VERSION declared in Makefile and
was still hardcoded. Fix this to improve version output and align to -h
output.

Signed-off-by: Christian Marangi <ansuelsmth@gmail.com>
---
 mreceive.c | 2 +-
 msend.c    | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

--- a/mreceive.c
+++ b/mreceive.c
@@ -93,7 +93,7 @@ int main(int argc, char *argv[])
 	ii = 1;
 
 	if ((argc == 2) && (strcmp(argv[ii], "-v") == 0)) {
-		printf("mreceive version 2.2\n");
+		printf("mreceive version %s\n", VERSION);
 		return 0;
 	}
 	if ((argc == 2) && (strcmp(argv[ii], "-h") == 0)) {
--- a/msend.c
+++ b/msend.c
@@ -102,7 +102,7 @@ int main(int argc, char *argv[])
 	int ret, i;
 
 	if ((argc == 2) && (strcmp(argv[ii], "-v") == 0)) {
-		printf("msend version 2.2\n");
+		printf("msend version %s\n", VERSION);
 		return 0;
 	}
 	if ((argc == 2) && (strcmp(argv[ii], "-h") == 0)) {
